tests/raw_to_standard/navjoy_568_test.py

```diff
@@ -0,0 +1,609 @@
+                },
+            ],
+            "streetNameFrom": "US-34",
+            "direction": "eastbound",
+            "requestedTemporarySpeed": "45",
+            "workStartDate": "2021-08-09T13:00:00.000Z",
+            "workEndDate": "2021-09-24T23:00:00.000Z",
+            "reductionJustification": "Crews roadside.",
+        }
+    }
+    assert navjoy_568.validate_closure(event) == True
+
+
+def test_validate_closure_missing_required_field_description():
+    event = {
+        "sys_gUid": "Form568-cb0fdaf0-c27a-4bef-aabd-442615dfb2d6",
+        "data": {
+            "srzmap": [
+                {
+                    "type": "LineString",
+                    "coordinates": [
+                        [
+                            [
+                                -103.17130040113868,
+                                40.625392709715676
+                            ],
+                            [
+                                -103.17889641706886,
+                                40.61979008921054
+                            ]
+                        ]
+                    ],
+                },
+            ],
+            "streetNameFrom": "US-34",
+            "directionOfTraffic": " East/West ",
+            "requestedTemporarySpeed": "45",
+            "workStartDate": "2021-08-09T13:00:00.000Z",
+            "workEndDate": "2021-09-24T23:00:00.000Z",
+            "reductionJustification": "Crews roadside.",
+        }
+    }
+    assert navjoy_568.validate_closure(event) == False
+
+
+def test_validate_closure_invalid_start_time():
+    event = {
+        "sys_gUid": "Form568-cb0fdaf0-c27a-4bef-aabd-442615dfb2d6",
+        "data": {
+            "descriptionForProject": "Bridge Repairs",
+            "srzmap": [
+                {
+                    "type": "LineString",
+                    "coordinates": [
+                        [
+                            [
+                                -103.17130040113868,
+                                40.625392709715676
+                            ],
+                            [
+                                -103.17889641706886,
+                                40.61979008921054
+                            ]
+                        ]
+                    ],
+                },
+            ],
+            "streetNameFrom": "US-34",
+            "directionOfTraffic": " East/West ",
+            "requestedTemporarySpeed": "45",
+            "workStartDate": 1713004011,
+            "workEndDate": "2021-09-24T23:00:00.000Z",
+            "reductionJustification": "Crews roadside.",
+        }
+    }
+    assert navjoy_568.validate_closure(event) == False
+
+
+def test_validate_closure_invalid():
+    event = 'invalid output'
+    assert navjoy_568.validate_closure(event) == False
+
+
+def test_validate_closure_no_data():
+    event = None
+    assert navjoy_568.validate_closure(event) == False
+
+
+def test_validate_closure_no_coordinates():
```

Comment:
This appears to have coordinates, albeit only a single point. What are we actually testing here?

Reply:
Updated the name to read test_validate_closure_invalid_coordinates